tools/scripts/post-build-cleanup.js

```diff
@@ -25,6 +25,15 @@ function prependSemicolon (fileName, text) {
     return fs.promises.readFile(`${buildDir}/${fileName}`, 'utf-8')
   }))
 
+  files.forEach((contents, i) => {
+    if (builtFileNames[i].includes('-loader') && builtFileNames[i].endsWith('.js')) {
+      const matches = contents.match(/\$&/)
```

Comment:
Good start and we can fortify this selector later for more problematic statements too